projects/plugins/jetpack/extensions/README.md

```diff
@@ -135,50 +133,33 @@ Since it's added to the beta array, you need to load the beta blocks as explaine
 
 ### Testing
 
-Run `pnpm test-extensions [--watch]` to run tests written in [Jest](https://jestjs.io/en/).
+Run `cd projects/plugins/jetpack && pnpm test-extensions [--watch]` to run tests written in [Jest](https://jestjs.io/en/).
 
 Note that adding [Jest snapshot tests](https://jestjs.io/docs/en/snapshot-testing) for block's `save` methods is problematic because many core packages relying on `window` that is not present when testing with Jest. See [prior exploration](https://github.com/Automattic/wp-calypso/pull/30727).
 
 #### Can I use Jurassic Ninja to test blocks?
 
 Yes! Just like any other changes in Jetpack, also blocks work in Jurassic Ninja.
 
-Simply add branch name to the URL: jurassic.ninja/create/?jetpack-beta&branches.jetpack=trunk or use other ninjastic features.
+Simply add branch name to the URL: jurassic.ninja/create/?jetpack-beta&branches.jetpack=master or use other ninjastic features.
```

Comment:
Really `master`, not `trunk`?

Reply:
The Jurassic Ninja interface hasn't been updated yet, so it does indeed still use the old slug, but the link will work.